userdocs/src/usage/nodegroup-override-announcement.md

```diff
@@ -0,0 +1,54 @@
+# Nodegroup Bootstrap Override For Custom AMIs
+
+This change was announced in the issue [Breaking: overrideBootstrapCommand soon...](https://github.com/weaveworks/eksctl/issues/3563).
+Now, it has come to pass in [this](https://github.com/weaveworks/eksctl/pull/4968) PR. Please read the attached issue carefully about 
+why we decided to move away from supporting custom AMIs without bootstrap scripts or with partial bootstrap scripts.
+
+We still provide a helper! Migrating hopefully is not that painful. `eksctl` still provides a helper script, which when
+sourced, will provide a couple of helpful environment properties and settings. This script is located [here](https://github.com/weaveworks/eksctl/blob/70a289d62e3c82e6177930cf2469c2572c82e104/pkg/nodebootstrap/assets/scripts/bootstrap.helper.sh).
```

Comment:
Is this script helpful? I cannot tell given you have said `helpX` only 3 times

Reply:
![seriously](https://user-images.githubusercontent.com/182850/160588203-66cf9ae8-da07-4629-bce7-c698243f28e8.jpeg)